Mage.Server/src/test/java/mage/server/util/Comparators.java

```diff
@@ -0,0 +1,115 @@
+package mage.server.util;
+
+import mage.server.util.config.GamePlugin;
+import mage.server.util.config.Plugin;
+import mage.server.util.config.Server;
+
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.Objects;
+
+public final class Comparators {
```

Comment:
These are just comparators used for testing. Nothing used in the main application.